internal/stats/blocks.go

```diff
@@ -84,7 +84,7 @@ func StartMonitoringBlocks(hactarClient *hactar.Client, lotusClient *lotus.Clien
 				return
 			case <-ticker.C:
 				log.Info("Block monitor ticked.")
-				SubmitNewBlockReport(hactarClient, lotusClient, currentSession)
+				// SubmitNewBlockReport(hactarClient, lotusClient, currentSession)
```

Comment:
Why commented out?

Reply:
Backend didn't have that endpoint implemented so it was always failing while I tested integration (I made a hotfix :stuck_out_tongue: clearly). Just didn't remember to undo this action...